BringgSDK.js

```diff
@@ -668,6 +675,30 @@ var BringgSDK = (function () {
     watchingOrder = isWatching;
   };
 
+  module._setUpConfigByToken = function(developerToken) {
+    // Check if this new token with region
+    if(!developerToken || developerToken.indexOf("_") === -1) {
```

Comment:
i think developerToken.indexOf("_") === -1 is not good enough as the tokn can be generated wil _ character regardless of the region at the start i.e 1234_abcsomesadsalad_.